keras-dreambooth-sprint/README.md

```diff
@@ -0,0 +1,155 @@
+
+![](https://huggingface.co/spaces/keras-dreambooth/README/resolve/main/dreambooth.png)
+
+**Welcome to Keras Dreambooth event!** 🤗 
+
+In this guide, we will walk you through what we will do in this event.
+
+We will be training dreambooth models using KerasCV and building demos on them. 
+
+**Getting Started** 🚀 
+
+To get started, join us in [hf.co/join/discord](http://hf.co/join/discord) in #keras-working-group channel. 
+
+We will be hosting our demos in this organization on Hugging Face Hub: [keras-dreambooth](https://huggingface.co/keras-dreambooth), send a request to join if you’d like to make a submission 🙂
```

Comment:
Here you can also share a direct join link https://huggingface.co/organizations/keras-dreambooth/share/vCakmJohtCHkocDUVWmjSHcMBiXeVsQoUo

Reply:
Not resolved still